src/land.c

```diff
@@ -999,7 +999,10 @@ void land_genWindows( int load, int changetab )
       if (planet_hasService(land_planet, PLANET_SERVICE_BAR))
          npc_generate(); /* Generate bar npc. */
    }
+   /* add to the ship log */
+   shiplog_append(0,land_planet->name);
```

Comment:
Why do we need this? This again seems to be useless info that just clutters up the log. Travel seems much too mundane to be recording.